### ISSUE TITLE
make colours more web accessible

### ISSUE THREAD

**Issue by donalmurtagh** [egeloen/travis-wall#9]
I haven't tested, but I suspect the colours used for the background/text would fail a [colour contrast accessibility test](http://webaim.org/resources/contrastchecker/).

The black on red for failed builds is particularly difficult to read. In my case the dashboard is displayed on a monitor about 3 meters from where I sit and it's difficult to read the text because the background and foreground colours are not sufficiently contrasting. 

Thanks for making this, it's great!

**Comment by egeloen** [comment]
Big :+1: the problem is the project is mostly broken due to BC breaks somewhere in the bower dependencies. I have already tried several time to fix it without success...

Anyway, thanks for the feedback I will fix that as soon as I'm able to fix dependencies...

**Comment by egeloen** [comment]
@donalmurtagh I have give a try to the tool you linked and mostly all background / text colors was passing except for the failling builds. I have lighten the red color in the referenced PR and I hope the UX would be better. If you think it is still not good enought, feel free to reopen the issue :)
